lang-lisp/chezscheme/autobuild/build

```diff
@@ -0,0 +1,13 @@
+abinfo "Configuring..."
+# parameters were taken from Arch Linux
+./configure --installprefix=/usr --threads \
```

Comment:
You should use "$SRCDIR" and not "./"

See: https://wiki.aosc.io/developer/packaging/package-styling-manual/